packages/driver/src/cy/assertions.coffee

```diff
@@ -56,6 +58,27 @@ parseValueActualAndExpected = (value, actual, expected) ->
 
   obj
 
+prepareObjsForDiff = (err) ->
+  if _.isString(err.actual) || _.isString(err.expected)
+    return err
+  ret = {}
+  ret.actual = mochaUtils.stringify(err.actual)
+  ret.expected = mochaUtils.stringify(err.expected)
+  return ret
+
+objToString = Object.prototype.toString
+
+_sameType = (a, b) ->
```

Comment:
is there a better way to check, maybe using some Lodash method?

Reply:
@bahmutov it's not a check its making sure the toString method is borrowed from Object so we don't accidentally call an overwritten toString on some custom object